docs/syscall_descriptions_syntax.md

```diff
@@ -129,8 +129,31 @@ structname "{" "\n"
 ```
 
 Fields can have attributes specified in parentheses after the field, independent
-of their type. The only attribute is direction (`in/out/inout`). For the field for
-which it is specified, the direction attributes on the upper levels are overridden.
+of their type. `in/out/inout` attribute specify per-field direction, for example:
+
+```
+foo {
+	field0	const[1, int32]	(in)
+	field1	int32		(inout)
+	field2	fd		(out)
+}
+```
+
+`out_overlay` attribute allows to have separate input and output layouts for the struct.
```

Comment:
Can syz-check adequately handle the structs with the `out_overlay` attribute?

Reply:
Good question. It doesn't. I will look into it.

Reply:
I will do this in a separate PR. This PR is large enough already and syz-check is not used systematically.